client/pom.xml

```diff
@@ -1,21 +1,18 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
+
   <parent>
     <groupId>com.microsoft.azure.simpletodo</groupId>
     <artifactId>simple-todo-parent</artifactId>
     <version>0.0.1-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
```

Comment:
are u able to delete this one?

Reply:
Yes. `../pom.xml` is default value. Delete it won't cause any problem.